.npmrc

```diff
@@ -1,2 +1,3 @@
-always-auth=true
 @xlts.dev:registry=https://registry.xlts.dev/
+//registry.xlts.dev/:always-auth=true
+//registry.xlts.dev/:_authToken=PASTE_AUTH_TOKEN_HERE
```

Comment:
```suggestion
```
We should just leave this blank. If the user has the token set up in their user `.npmrc` file or if they've (in our case) logged in via the npm CLI, then this isn't needed.